projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-filtering.component.ts

```diff
@@ -509,6 +509,11 @@ export class IgxGridExcelStyleFilteringComponent extends BaseFilteringComponent
     private renderValues() {
         this.filterValues = this.generateFilterValues();
         this.generateListData();
+        this.expressionsList.forEach(expr => {
+            if (expr.expression.condition.name === 'in' && this.column.dataType === GridColumnDataType.String) {
```

Comment:
I don't think we need the condition check but we may want to check the column's filteringIgnoreCase prop (because if false, the modification is not needed).
```suggestion
            if (this.column.dataType === GridColumnDataType.String && this.column.filteringIgnoreCase) {
```